### PR TITLE
Issue 693: Fix Submit button not responding

### PR DESCRIPTION
**Pull request by kevinherdez**
This an issue that seems to be what's causing this issue #693 

On a clean launch, it doesn't seem like the navigator for the Observations screens are loaded until the user goes to that tab. If a user goes to a forecast page, without opening that tab, and then hits the Submit button, the app can't navigate to that screen and React Native throws this error:

`The action 'NAVIGATE' with payload {"name":"observationSubmit","params":{"center_id":"NWAC"}} was not handled by any navigator.`

This fix adds the `ObservationSubmitScreen` to the `AvalancheCenterStack.Navigator` in the `HomeScreen`

*Import thing to note*: This changes the behavior so that if the Submit button is tapped on a forecast screen, the `ObservationSubmitScreen` is added to the home screen stack which keeps the user in the Zones tab. It no longer changes to the Observations tab

| Before | After |
|---|---|
|SubmitBefore|SubmitAfter|

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,5 @@ android
 # Jetbrains IDE files
 .idea
 
+# Ignore changes to the package-lock
```
Comment (kevinherdez) [inline .gitignore:24]: I added this because I noticed that the `package-lock.json` wasn't being tracked

File: components/screens/ObservationsScreen.tsx
Comment (yuliadub) [inline components/screens/ObservationsScreen.tsx:1]: doesnt look anything got changed in this file, just formatting? that sound right?

**Comment by yuliadub** [comment]
nice catch - your fix makes sense to me! I can 100% repro what you are describing in expo, the preview and the production app (the submit button doesn't work for me if I go to the zone first vs loading the main obs screen from the main without a zone)

looks like we got some formatting spacing differences (I noticed this too when I recently submitted a PR after opening the files from this repo in Vscode). @stevekuznetsov will be back next week so we can decide if we want to revert those or maybe do a change for the whole repo.... I dont feel too strongly one way or another

**Comment by kevinherdez** [comment]
> nice catch - your fix makes sense to me! I can 100% repro what you are describing in expo, the preview and the production app (the submit button doesn't work for me if I go to the zone first vs loading the main obs screen from the main without a zone)
> 
> looks like we got some formatting spacing differences (I noticed this too when I recently submitted a PR after opening the files from this repo in Vscode). @stevekuznetsov will be back next week so we can decide if we want to revert those or maybe do a change for the whole repo.... I dont feel too strongly one way or another

I didn't realize that there was CI checking for this, so I ran `yarn prettify` and it reverted those changes. It seems like there's a decision that was made here already on the styling, and I'm not familiar enough with JS to know which one is "standard" between what I have on VS Code and what `prettier` is doing, so I'll just keep using `prettier` for now

**Comment by yuliadub** [comment]
> > nice catch - your fix makes sense to me! I can 100% repro what you are describing in expo, the preview and the production app (the submit button doesn't work for me if I go to the zone first vs loading the main obs screen from the main without a zone)
> > looks like we got some formatting spacing differences (I noticed this too when I recently submitted a PR after opening the files from this repo in Vscode). @stevekuznetsov will be back next week so we can decide if we want to revert those or maybe do a change for the whole repo.... I dont feel too strongly one way or another
> 
> I didn't realize that there was CI checking for this, so I ran `yarn prettify` and it reverted those changes. It seems like there's a decision that was made here already on the styling, and I'm not familiar enough with JS to know which one is "standard" between what I have on VS Code and what `prettier` is doing, so I'll just keep using `prettier` for now

sounds good! I (in all honesty) am really bad and dont have pre commit hooks set up (I really should) so I always forget and screw up styling as well. But agreed that sticking to what the repo has for now makes the most sense!

**Comment by stevekuznetsov** [comment]
Thanks for the fix! Love to see it. I don't know there is any standard in JS for formatting so each project is free to configure their `.prettierrc`. You should be able to configure VSCode to honor it as well! I am surprised it does not do so by default.

**Pull request merged**